.gitignore

```diff
@@ -20,3 +20,5 @@ examples/tensorflow/distributed_mnist/data
 doc/_build
 **/.DS_Store
 venv/
+*.rec
+*~
```

Comment:
What is this symbol?

Reply:
It was added from a sync pull I made on the original repo. Not mine.